README.md

```diff
@@ -243,23 +243,21 @@ CA Cluster Issuer Sample `values.yaml`
 
 Harbor is an open source trusted cloud native registry project that stores, signs, and scans content. Harbor extends the open source Docker Distribution by adding the functionalities usually required by users such as security, identity and management. Having a registry closer to the build and run environment can improve the image transfer efficiency. Harbor supports replication of images between registries, and also offers advanced security features such as user management, access control and activity auditing.
 
-- The [contour package](#contour) is a prerequisite for Harbor v2.9.x, so that must be installed
-  first if you are using v2.9.x.
-- Harbor v2.11.2 supports exposing the registry using loadbalancer, and contour is not required.
+- Since v2.11.2, Harbor supervisor service supports exposing the registry using loadbalancer, and contour is not required.
```

Comment:
nit: I'd suggest stating `Contour is optional, depending on the configuration settings` rather than `contour is not required` which implies that it should never be used.

Reply:
Also: loadbalancer- > a load balancer